torch/nn/attention/flex_attention.py

```diff
@@ -108,13 +117,14 @@ def _identity(
     return score
 
 
-def _no_mask(
+def noop_mask(
```

Comment:
I'm a bit confused... Is `noop_mask` a Tensor or a mask_mod callable?

Reply:
It's the default ```mask_mod``` callable.